python/py-bitarray/Portfile

```diff
@@ -35,13 +35,7 @@ if {${name} ne ${subport}} {
         xinstall -m 0644 -W ${worksrcpath} README.rst ${destroot}${docdir}
     }
 
-    pre-test {
-        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
-    }
-
     test.run        yes
```

Comment:
do the tests run in this way? I would think you have to set `python.test_framework` to an empty string as otherwise it will assume `pytest` and add `test.args` that will probably now work with the way tests are called here. Removing `test.target` should be fine though.

Reply:
@reneeotten yes, I’ve run tests this way

Reply:
```
--->  Computing dependencies for py311-bitarray.
--->  Testing py311-bitarray
Executing:  cd "/opt/local/var/macports/build/_Users_catap_src_macports-ports_python_py-bitarray/py311-bitarray/work/bitarray-2.8.1" && /opt/local/Library/Frameworks/Python.framework/Versions/3.11/bin/python3.11 bitarray/test_bitarray.py -o addopts='' 
/opt/local/var/macports/build/_Users_catap_src_macports-ports_python_py-bitarray/py311-bitarray/work/bitarray-2.8.1/bitarray/test_bitarray.py:5057: DeprecationWarning: unittest.makeSuite() is deprecated and will be removed in Python 3.13. Please use unittest.TestLoader.loadTestsFromTestCase() instead.
  suite.addTest(unittest.makeSuite(cls))
............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 492 tests in 0.666s

OK
bitarray is installed in: /opt/local/var/macports/build/_Users_catap_src_macports-ports_python_py-bitarray/py311-bitarray/work/bitarray-2.8.1/bitarray
bitarray version: 2.8.1
sys.version: 3.11.5 (main, Aug 25 2023, 01:35:05) [Clang 13.1.6 (clang-1316.0.21.2.5)]
sys.prefix: /opt/local/Library/Frameworks/Python.framework/Versions/3.11
pointer size: 64 bit
sizeof(size_t): 8
sizeof(bitarrayobject): 80
__clang__ or __GNUC__ defined: 1
PY_LITTLE_ENDIAN (use word shift): 1
DEBUG: 0
```

Reply:
But idea to add `python.test_framework` with empty string is good one, it allows to remove useless dependency, let me do it.

Reply:
here it is

Reply:
thanks, you can also see that without that addition it added "-o addopts=" which is a outest flag that shouldn't be added here.